core/contracts/ContractCreator.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import "./Finder.sol";
+import "./Registry.sol";
+
+
+// TODO(ptare): Make this (and all contracts) Withdrawable.
+contract ContractCreator {
+    address internal finderAddress;
+    address internal adminAddress;
+    address internal priceFeedAddress;
```

Comment:
This looks like copy/pasta from `v0`, but I'm having a hard time understanding why `adminAddress` and `priceFeedAddress` might belong in this contract, and what this contract's responsibilities are at a high level.